static/docs/commands-reference/import.md

```diff
@@ -72,3 +73,47 @@ force updating the downloaded data artifact from the external DVC repo.
   problems arise, otherwise 1.
 
 - `-v`, `--verbose` - displays detailed tracing information.
+
+## Example
+
+An obvious case for this command is to import a complete ML model from an
```

Comment:
in this case it's better to pull dataset - data.xml from the repo

Reply:
we'll need to come up with a better example repo for this when we start doing use cases and updating get started

Reply:
changed to download dataset instead in d739685.

I agree it would be nice to have other repo examples. What do you have in mind? I'm almost ready to start updating the _Get Started_ and _Use Cases_.

Reply:
Updated requirements in https://github.com/iterative/dvc.org/issues/487#issuecomment-517520838